src/main/java/appeng/core/sync/packets/PacketPartPlacement.java

```diff
@@ -20,58 +20,70 @@
 
 
 import io.netty.buffer.ByteBuf;
-import io.netty.buffer.Unpooled;
 
-import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.entity.player.EntityPlayerMP;
 
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
+
 import appeng.core.CommonHelper;
 import appeng.core.sync.AppEngPacket;
-import appeng.core.sync.network.INetworkInfo;
+import appeng.core.sync.AppEngPacketHandler;
 import appeng.parts.PartPlacement;
 
 
-public class PacketPartPlacement extends AppEngPacket
+public class PacketPartPlacement implements AppEngPacket, AppEngPacketHandler<PacketPartPlacement, AppEngPacket>
 {
 
-	int x;
-	int y;
-	int z;
-	int face;
-	float eyeHeight;
+	private int x;
+	private int y;
+	private int z;
+	private int face;
+	private float eyeHeight;
 
 	// automatic.
-	public PacketPartPlacement( ByteBuf stream )
+	public PacketPartPlacement()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)